client/src/App.js

```diff
@@ -3,14 +3,17 @@ import React from "react";
 import { Hikes } from "./components/Hikes";
 import { MyMap } from "./components/maps/MyMap.js";
 import { Form } from "./components/Form.js";
-
 require("dotenv").config();
 
 export default function App() {
 
   return (
     <div className ='app'>
-      <h1>Pin My Hike</h1> 
+      <div className="title-overlay">
+                <img className="PMH-logo" 
```

Comment:
indentation 😱

Reply:
Yus, will no fix immediately. Thoughts of PR has brought about complacency.. 1000 apologies